mod_reforged/hooks/ui/screens/tooltip/tooltip_events.nut

```diff
@@ -1,6 +1,157 @@
+local generateExtendedAttributeStats = function( _entityId, _elementId, _elementOwner )
```

Comment:
This function is only used down below inside another function _once_. Therefore, having it as a local function doesn't really benefit anything except perhaps readability of the other function. Because it is local, no submodder can hook it and change anything. I'd suggest making it a proper new function inside that class so that modders could tweak it if they want.